### PR TITLE
chore: add terraform-google-gen-ai-document-summarization repo

### PR DESCRIPTION
**Pull request by yil532**
Onboarding https://github.com/GoogleCloudPlatform/terraform-google-gen-ai-document-summarization. This repo hosts UP16.1.

### CODE REVIEW

**Review by nicain — COMMENTED** [review]
LGTM

**Review by bharathkkb — APPROVED** [review]

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @yil532. Looks like @g-awmalik had some additional comments in b/289379695#comment2 around repo naming which would be good to resolve before we apply this.

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -711,5 +711,14 @@ locals {
       owners      = ["Saipriyavk", "ChrisBarefoot"]
       topics      = local.common_topics.net
     },
+    {
+      name            = "terraform-google-gen-ai-document-summarization"
+      short_name      = "gen-ai-document-summarization"
+      org             = "GoogleCloudPlatform"
+      description     = "Summarizes document using OCR and Vertex Generative AI LLM"
+      owners          = ["asrivast", "balajismaniam", "erschmid", "liyvonne", "nicain"]
```
Comment (bharathkkb) [inline infra/terraform/test-org/org/locals.tf:719]: Please use github usernames

```suggestion
      owners          = ["asrivas", "balajismaniam", "telpirion", "yil532", "nicain"]
```

**Comment by apeabody** [comment]
@bharathkkb - For lint: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/1693

**Comment by balajismaniam** [comment]
The repo has been renamed to https://github.com/GoogleCloudPlatform/terraform-genai-doc-summarization. CC: @bharathkkb

**Comment by bharathkkb** [comment]
Thanks @balajismaniam

File: infra/terraform/test-org/org/locals.tf [hunk 2]
```diff
@@ -711,5 +711,14 @@ locals {
       owners      = ["Saipriyavk", "ChrisBarefoot"]
       topics      = local.common_topics.net
     },
+    {
+      name            = "terraform-google-gen-ai-document-summarization"
```
Comment (bharathkkb) [inline infra/terraform/test-org/org/locals.tf:715]:
```suggestion
      name            = "terraform-genai-doc-summarization"
```

File: infra/terraform/test-org/ci-triggers/locals.tf [hunk 3]
```diff
@@ -27,19 +27,20 @@ locals {
...
+    "dynamic-python-webapp"         = "terraform-dynamic-python-webapp"
+    "dynamic-javascript-webapp"     = "terraform-dynamic-javascript-webapp"
+    "deploy-java-multizone"         = "terraform-example-deploy-java-multizone"
+    "ecommerce-microservices"       = "terraform-ecommerce-microservices-on-gke"
+    "deploy-java-gke"               = "terraform-example-deploy-java-gke"
+    "large-data-sharing-java-app"   = "terraform-large-data-sharing-java-webapp"
+    "large-data-sharing-go-app"     = "terraform-large-data-sharing-golang-webapp"
+    "java-dynamic-point-of-sale"    = "terraform-example-java-dynamic-point-of-sale"
+    "ml-image-annotation-gcf"       = "terraform-ml-image-annotation-gcf"
+    "gen-ai-document-summarization" = "terraform-google-gen-ai-document-summarization"
```
Comment (bharathkkb) [inline infra/terraform/test-org/ci-triggers/locals.tf:43]:
```suggestion
    "gen-ai-document-summarization" = "terraform-genai-doc-summarization"
```

**Review by bharathkkb — APPROVED** [review]

File: infra/terraform/test-org/org/locals.tf [hunk 4]
```diff
@@ -711,5 +711,14 @@ locals {
       owners      = ["Saipriyavk", "ChrisBarefoot"]
       topics      = local.common_topics.net
     },
+    {
+      name            = "terraform-genai-doc-summarization"
+      short_name      = "gen-ai-document-summarization"
```
Comment (bharathkkb) [inline infra/terraform/test-org/org/locals.tf:716]:
```suggestion
      short_name      = "genai-doc-summarization"
```

File: infra/terraform/test-org/ci-triggers/locals.tf [hunk 5]
```diff
@@ -27,19 +27,20 @@ locals {
...
+    "dynamic-python-webapp"         = "terraform-dynamic-python-webapp"
+    "dynamic-javascript-webapp"     = "terraform-dynamic-javascript-webapp"
+    "deploy-java-multizone"         = "terraform-example-deploy-java-multizone"
+    "ecommerce-microservices"       = "terraform-ecommerce-microservices-on-gke"
+    "deploy-java-gke"               = "terraform-example-deploy-java-gke"
+    "large-data-sharing-java-app"   = "terraform-large-data-sharing-java-webapp"
+    "large-data-sharing-go-app"     = "terraform-large-data-sharing-golang-webapp"
+    "java-dynamic-point-of-sale"    = "terraform-example-java-dynamic-point-of-sale"
+    "ml-image-annotation-gcf"       = "terraform-ml-image-annotation-gcf"
+    "gen-ai-document-summarization" = "terraform-genai-doc-summarization"
```
Comment (bharathkkb) [inline infra/terraform/test-org/ci-triggers/locals.tf:43]:
```suggestion
    "genai-doc-summarization" = "terraform-genai-doc-summarization"
```

**Comment by bharathkkb** [comment]
This had been applied

**Pull request merged**